source/install/install-kubernetes-mattermost.rst

```diff
@@ -107,3 +105,123 @@ Use your domain registration service to create a canonical name or IP address re
+
+Save the file as ``secret.yaml``. The example below is for AWS/S3.
+
+.. code-block:: yaml
+
+   apiVersion: v1
+   kind: Secret
+   metadata:
+    name: test-restore
+   type: Opaque
+   stringData:
+    AWS_ACCESS_KEY_ID: XXXXXXXXXXXX
+    AWS_SECRET_ACCESS_KEY: XXXXXXXXXXXX/XXXXXXXXXXXX
+    AWS_REGION: us-east-1
+    S3_PROVIDER: AWS
+
+**Parameters**
+- ``name``. The name of this manifest which is referenced in the installation manifest.
+
+
+6. Create a Mattermost cluster installation manifest.
+
+Open a text editor and create a text file with the following details. Save the file as ``mattermost-installation.yaml``:
+
+.. code-block:: yaml
+
+  apiVersion: mattermost.com/v1alpha1
+  kind: ClusterInstallation
+  metadata:
+    name: mm-example-full
+  spec:
+    size: ""
+    ingressName: example.mattermost-example.com
+    ingressAnnotations:
+      kubernetes.io/ingress.class: nginx
+    version: 5.14.0
+    mattermostLicenseSecret: ""
+    database:
+      storageSize: 50Gi
+    minio:
+      storageSize: 50Gi
+    elasticSearch:
+      host: ""
+      username: ""
+      password: ""
+
+The Mattermost installation manifest contains fields which must be edited in line with your configuration and environment requirements.
+
+.. csv-table::
+    :header: "Field", "Description", "Must Edit"
+
+    "metadata.name", "The name of your Mattermost as it will be shown in Kubernetes. The shorter the better.", "Yes"
+    "spec.size", "The size of your installation. This can be '100users', '1000users, '5000users', '10000users', or '25000users' Custom values are accepted provided the replication formula is correct.", "Yes"
+    "spec.ingressName", "The DNS for your Mattermost installation. This is the same name as the existing deployment, but the underlying resource is the restored database.", "Yes"
+    "spec.version", "The Mattermost version.", "No"
+    "spec.mattermostLicenseSecret", "The name of the Kubernetes secret containing your license (e.g. mattermost-license). Required for Enterprise deployments.", "Yes"
+    "spec.database.storageSize", "The storage size for your database. Your Kubernetes cluster must have volumes this size or larger.", "No"
+    "spec.minio.storageSize", "The storage size for your file storage. Your Kubernetes cluster must have volumes this size or larger.", "No"
+    "spec.elasticSearch", "The section for Elasticsearch settings. Remove this section if you will not be using Elasticsearch.", "No"
+    "spec.elasticSearch.host", "The hostname for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch.", "No"
+    "spec.elasticSearch.username", "The username for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch.", "No"
+    "spec.elasticSearch.password", "The password for your Elasticsearch instance. Remove this section if you will not be using Elasticsearch.", "No"
```

Comment:
This looks to be the same table as above. Do we want to duplicate it? 0/5 on this, but would need to change the license column to match above if that is changed.

Reply:
Do we want to change the license secret name or remove reference? Happy to remove the table too, also 0/5 on that.

Reply:
I am also 0/5 on it, but my slight preference would be to remove the table just to not have two things that could possibly be out of sync.